pluginlib/pluginlib-extras.cmake

```diff
@@ -26,6 +26,12 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "Clang" AND CMAKE_CXX_FLAGS MATCHES "-stdlib=li
   else()
     set(FILESYSTEM_LIB c++fs)
   endif()
+elseif(CMAKE_CXX_COMPILER_ID MATCHES "QCC")
+  if(CMAKE_CXX_FLAGS MATCHES "-stdlib=libc\\+\\+")
+    set(FILESYSTEM_LIB c++fs)
+  else()
+    set(FILESYSTEM_LIB)
+  endif
```

Comment:
Syntax error appears to have broken all Dashing builds: http://build.ros2.org/view/Dci/job/Dci__nightly-release_ubuntu_bionic_amd64/422/console

Fixed by https://github.com/ros/pluginlib/pull/208